target/classes/application.properties

```diff
@@ -0,0 +1,7 @@
+spring.datasource.url=jdbc:postgresql://ec2-54-235-119-42.compute-1.amazonaws.com
```

Comment:
I see what you're trying to do here but this stuff should never be in GitHub. Heroku will give you all of this at runtime on production. If you are still struggling with Heroku postgres deployments let's review it.